bundle/edu.gemini.itc.shared/src/main/Java/edu/gemini/itc/shared/ObservationDetails.java

```diff
@@ -1,9 +1,13 @@
-package edu.gemini.itc.service;
+package edu.gemini.itc.shared;
+
+import edu.gemini.itc.shared.*;
+
+import java.io.Serializable;
 
 /**
  * Container for observation detail parameters.
  */
-public final class ObservationDetails {
+public final class ObservationDetails implements Serializable {
```

Comment:
Yes, this are the TOs passed around using TRPC, so they all have to be serializable.